src/ui/pages/BlogPage/Editor/Editor.jsx

```diff
@@ -76,7 +78,8 @@ export class RichTextEditor extends React.Component {
       return true;
     });
 
-    this.setState({ cats: await fetchAllCategories() });
+    const user = await fetchUserInfo()
+    this.setState({ cats: await fetchAllCategories(), author: user.user.displayName});
```

Comment:
maybe we should move the gateway call to a const one line up. It would be easier to read imo. What do you think?